packages/ui/src/components/va-tree-view/VaTreeView.demo.vue

```diff
@@ -1,59 +1,391 @@
 <template>
   <VbDemo>
-    <VbCard>
-      <va-button @click="$refs.treeView.expand()">
-        EXPAND ALL
-      </va-button>
-      <va-button @click="$refs.treeView.collapse()">
-        COLLAPSE ALL
-      </va-button>
-
-      <va-tree-view ref="treeView">
-        <va-tree-category
-          label="Category text"
-          is-open
-        >
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-          <va-tree-category
-            label="Category text"
-            is-open
-          >
-            <va-tree-node>
-              Node text
-            </va-tree-node>
-          </va-tree-category>
-        </va-tree-category>
-      </va-tree-view>
+    <VbCard title="Default">
+      <va-tree-view :nodes="nodes" />
+    </VbCard>
+    <VbCard title="Stateless">
+      <p>
+        expansion/selection should work:
+        <br />
+        checked model: {{stateChecked}}
+        <br />
+        expanded model: {{stateExpanded}}
+      </p>
+      <va-tree-view
+        v-model:checked="stateChecked"
+        v-model:expanded="stateExpanded"
+        :stateful="false"
+        :nodes="nodes"
+        selectable
+      />
+      <p>
+        expansion/selection shouldn't work:
+      </p>
+      <va-tree-view
+        :stateful="false"
+        :nodes="nodes"
+        selectable
+      />
+    </VbCard>
+    <VbCard title="Selectable">
```

Comment:
<img width="486" alt="image" src="https://user-images.githubusercontent.com/5394573/193349787-ed4fee49-e27d-4039-88df-91e669a63de5.png">

Selection model demo has more stuff. We don't need both.